src/lib/drupal/listingPages.ts

```diff
@@ -15,8 +15,15 @@ const LISTING_RESOURCE_TYPES = [
   RESOURCE_TYPES.EVENT_LISTING,
 ] as const
 
+const SINGLE_PAGE_LISTING_RESOURCE_TYPES = [
```

Comment:
Can we document why these are distinct from other listing pages? A comment here with a simple explanation and a link to a README with some more detail would be great.